ecr/base.go

```diff
@@ -51,39 +52,75 @@ type ecrAPI interface {
 	PutImageWithContext(aws.Context, *ecr.PutImageInput, ...request.Option) (*ecr.PutImageOutput, error)
 }
 
-func (b *ecrBase) getManifest(ctx context.Context) (*ecr.Image, error) {
-	imageIdentifier := b.ecrSpec.ImageID()
-	log.G(ctx).WithField("imageIdentifier", imageIdentifier).Debug("ecr.base.manifest")
-	batchGetImageInput := &ecr.BatchGetImageInput{
-		RegistryId:     aws.String(b.ecrSpec.Registry()),
-		RepositoryName: aws.String(b.ecrSpec.Repository),
-		ImageIds:       []*ecr.ImageIdentifier{imageIdentifier},
-		// TODO: Determine if this should be hard-coded
-		AcceptedMediaTypes: []*string{
-			aws.String(ocispec.MediaTypeImageManifest),
-			aws.String(images.MediaTypeDockerSchema2Manifest),
+// getImage fetches the reference's image from ECR.
+func (b *ecrBase) getImage(ctx context.Context) (*ecr.Image, error) {
+	return b.runGetImage(ctx, ecr.BatchGetImageInput{
+		ImageIds: []*ecr.ImageIdentifier{b.ecrSpec.ImageID()},
+	})
```

Comment:
Note to self: even though the accepted media types are removed in this commit, they are added back in [`4bf7877` (#13)](https://github.com/awslabs/amazon-ecr-containerd-resolver/pull/13/commits/4bf7877a8b124c30c090285fffce4e8e03c8cc4b).

Reply:
I fixed this so that the removal is atomic across the commits.